geometry_msgs/PoseArray initial_poses
nav_msgs/OccupancyGrid map
float32 tool_width
---
bool error
nav_msgs/OccupancyGrid coverage_map
nav_msgs/Path[] coverage_paths